ComfySigns/ComfySigns.cs

```diff
@@ -110,5 +116,12 @@ public class ComfySigns : BaseUnityPlugin {
         }
       }
     };
+    public static void AddFallbackFont(TMP_FontAsset font) {
+      if (font.fallbackFontAssetTable != null) {
+        font.fallbackFontAssetTable.Add(UIFonts.GetFontAsset("Norse SDF"));
+      } else {
+        font.fallbackFontAssetTable = new List<TMP_FontAsset>() { UIFonts.GetFontAsset("Norse SDF") };
```

Comment:
This will add the same FontAsset repeatedly to the List (since it's not a Set) so it might bloat based on how many config changes trigger this method.

I'd modify the method to take an additional argument: `TMP_FontAsset fallbackFont`

```
if (font.fallbackFontAssetTable == null) {
  font.fallbackFontAssetTable = new List<TMP_FontAsset>() { fallbackFont };
} else if (!font.fallbackFontAssetTable.Contains(fallbackFont)) {
  font.fallbackFontAssetTable.Add(fallbackFont);
}
```